helm/nessie/templates/hpa.yaml

```diff
@@ -17,12 +23,24 @@ spec:
     - type: Resource
       resource:
         name: cpu
+        {{- if .Capabilities.APIVersions.Has "autoscaling/v2beta1" -}}
```

Comment:
Is it not cumulative? I.e. does `Has v2` imply `has v2beta1`?... or not?

Reply:
Whoops you are right :-)